ui/src/views/Tasks/TaskIndex/IndexedTask/index.jsx

```diff
@@ -98,14 +97,11 @@ export default class IndexedTask extends Component {
         error: indexedTaskError,
         loading: indexedTaskLoading,
       },
-      description,
     } = this.props;
     const loading = latestArtifactsLoading || indexedTaskLoading;
 
     return (
-      <Dashboard
-        title="Index Browser"
-        helpView={<HelpView description={description} />}>
```

Comment:
OH, I just realized that we did have a description set in https://github.com/taskcluster/taskcluster/blob/67af2aff22519eb0433a0e081efba8d512cb90ed/ui/src/views/Tasks/routes.js#L38-L39

but for some reason, it wasn't reaching the views. Let's make sure that description is shown on both views. I think we will need to modify `TaskIndex/routes.js` to have the description. Apologies for not seeing that earlier.

Reply:
Oh, really. Done!)